draft-ietf-scitt-architecture.md

```diff
@@ -401,16 +401,19 @@ This specification leaves implementation, encoding and documentation of Registra
 
 #### Mandatory Registration Checks
 
-During registration, a Transparency Service MUST, at a minimum, authenticate the Issuer of the Signed Statement by validating the COSE signature and checking the identity of the issuer against one of its currently configured trust anchors, using the `x5t` (34), `x5chain`(33) or `kid`(4) protected headers of the Signed Statement as hints.
-For instance, in order to authenticate X.509 Signed Statements, the Transparency Service MUST build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
+During registration, a Transparency Service MUST, at a minimum, syntactically check the Issuer of the Signed Statement by validating the COSE signature and checking the Issuer details are consistent with one of its supported trust anchors.
+The Issuer identity MUST be bound to the Signed Statement by including a commitment to it in the protected header.
+If the protected header includes multiple commitments, all those that are registered by the Transparency Service MUST be checked.
+
+For instance, when using X.509 Signed Statements, the Transparency Service MUST build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
 
 The Transparency Service MUST apply the Registration Policy that was most recently added to the Append-only Log at the time of registration.
 
 #### Auditability of Registration
 
 The operator of a Transparency Service MAY update the Registration Policy or the trust anchors of a Transparency Service at any time.
 
-Transparency Services MUST ensure that for any Signed Statement they register, enough information is made available to Auditors (either in the Append-only Log and retrievable through audit APIs, or included in the Receipt) to authenticate and retrieve the Transparent Statements describing the registration policy and trust anchors that apply to this registration.
+Transparency Services MUST ensure that for any Signed Statement they register, enough information is made available to Auditors (either in the Append-only Log and retrievable through audit APIs, or included in the Receipt) to reproduce the Registration checks that were defined by the Registration Policies at the time of Registration.
```

Comment:
If we do not define in effective terms what a registration policy is and what are its minimal mandatory elements (not arguing with this I agree with that for now) generally, how can we make this change a MUST?

Reply:
yep, good point. i would like it to be a SHOULD

Reply:
I believe this is threading the needle, saying a registered statement must have enough info for it to be auditable, while leaving the details to be implementation-specific. Otherwise, what's the point? 
The [Registration Policies section](https://ietf-wg-scitt.github.io/draft-ietf-scitt-architecture/draft-ietf-scitt-architecture.html#name-registration-policies) defines this:
> This specification leaves implementation, encoding and documentation of Registration Policies and trust anchors to the operator of the Transparency Service.

Reply:
Yes, that was exactly the intent

Reply:
Others are free to approve, I will not approve without the change below.

```suggestion
Transparency Services SHOULD ensure that for any Signed Statement they register, enough information is made available to Auditors (either in the Append-only Log and retrievable through audit APIs, or included in the Receipt) to reproduce the Registration checks that were defined by the Registration Policies at the time of Registration.
```

If not, I expect a cross-linked PR to explicate all the registration policy specification that was removed in the past (for lack of alignment and ambiguity and generalities) and explicate it all. The WG considered this not ideal in previous editorial review, so I assume this alternative approach is unlikely. So I would change it to SHOULD.

Reply:
@aj-stein-nist, I agree the MUST/SHOULD warrants some more discussion. Purely from a focus of the changes in this PR perspective, can we open a separate PR for tracking the normative references without more clarity on details?